dwave/system/samplers/leap_hybrid_sampler.py

```diff
@@ -87,19 +87,23 @@ class LeapHybridSampler(dimod.Sampler):
 
     def __init__(self, solver=None, connection_close=True, **config):
 
+        # always use the base class (QPU client filters-out the hybrid solvers)
+        config['client'] = 'base'
```

Comment:
Well if you insist I review: "client filters-out the" --> "client filters out the"

Reply:
You are the author of `LeapHybridSampler`, after all. :smiley:

Reply:
But, hope you'll excuse me if I do not produce a separate commit to remove a single dash. :smile:

Reply:
Sure, you know I'll get blamed for it.

Reply:
My favourite commit of all time https://github.com/dwavesystems/dimod/pull/619/commits/cea9af79d6ddc64fe813d8d016e5696aa74bef5f